app/styles/base/banner.scss

```diff
@@ -4,19 +4,25 @@ bl-banner {
 }
 
 bl-banner > md-card.mat-card {
```

Comment:
can you use this as an opportunity to move the `banner.scss` file in the same folder as the component and have the import "./banner.scss" in the component